src/react-native/rollbar.js

```diff
@@ -23,7 +23,12 @@ function Rollbar(options, client) {
   delete this.options.maxItems;
   this.options.environment = this.options.environment || 'unspecified';
   var api = new API(this.options, transport, urllib);
-  this.client = client || new Client(this.options, api, logger, 'react-native');
+
+  // remove tracer object from options so we don't try to parse it
+  var tracer = options.tracer || null;
+  delete options.tracer;
```

Comment:
Each of browser, server, and react-native does this the same way. This could be moved into the `Client` constructor, keeping it in one place, and then the additional argument to the constructor isn't needed either.